services/src/handlers/wfs.rs

```diff
@@ -228,7 +228,9 @@ async fn get_feature<C: Context>(
             let time = TimeInstance::from(chrono::offset::Utc::now());
             TimeInterval::new_unchecked(time, time)
         }),
-        spatial_resolution: SpatialResolution::zero_point_one(),
+        spatial_resolution: request
+            .query_resolution
+            .unwrap_or_else(SpatialResolution::zero_point_one),
```

Comment:
is this is meaningful default value? Should id maybe be dependant of the projection? Maybe put a TODO here to check this

Reply:
f820d8f